doc/api/http2.md

```diff
@@ -1714,6 +1714,14 @@ changes:
   * `Http1ServerResponse` {http.ServerResponse} Specifies the ServerResponse
     class to used for HTTP/1 fallback. Useful for extending the original
     `http.ServerResponse`. **Default:** `http.ServerResponse`
+  * `Http2ServerRequest` {http2.Http2ServerRequest} Specifies the
+    Http2ServerRequest class to used.
```

Comment:
One final comment as I was going through this: `to use.` or `to be used.` Same below.